src/corelocation.cs

```diff
@@ -1088,4 +1126,167 @@ interface CLLocationSourceInformation : NSCopying, NSSecureCoding {
 		bool IsProducedByAccessory { get; }
 	}
 
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	interface CLUpdate {
+		[Export ("isStationary")]
+		bool IsStationary { get; }
+
+		[NullAllowed, Export ("location")]
+		CLLocation Location { get; }
+	}
+
+	[NoWatch, NoTV, Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface CLMonitoringRecord : NSSecureCoding {
+		[Export ("condition", ArgumentSemantic.Strong)]
+		CLCondition Condition { get; }
+
+		[Export ("lastEvent", ArgumentSemantic.Strong)]
+		CLMonitoringEvent LastEvent { get; }
+	}
+
+	[NoWatch, NoTV, Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface CLMonitoringEvent : NSSecureCoding {
+		[Export ("identifier", ArgumentSemantic.Strong)]
+		string Identifier { get; }
+
+		[NullAllowed, Export ("refinement", ArgumentSemantic.Strong)]
+		CLCondition Refinement { get; }
+
+		[Export ("state")]
+		CLMonitoringState State { get; }
+
+		[Export ("date", ArgumentSemantic.Strong)]
+		NSDate Date { get; }
+	}
+
+	[NoWatch, NoTV, Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	interface CLMonitorConfiguration {
+		[Export ("name")]
+		string Name { get; }
+
+		[Export ("queue")]
+		DispatchQueue Queue { get; }
+
+		[Export ("eventHandler")]
+		Action<CLMonitor, CLMonitoringEvent> EventHandler { get; }
+
+		[Static]
+		[Export ("configWithMonitorName:queue:eventHandler:")]
+		CLMonitorConfiguration Create (string name, DispatchQueue queue, Action<CLMonitor, CLMonitoringEvent> eventHandler);
+	}
+
+	[NoWatch, NoTV, Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface CLMonitor {
+		[Async]
+		[Static]
+		[Export ("requestMonitorWithConfiguration:completion:")]
+		void RequestMonitor (CLMonitorConfiguration config, Action<CLMonitor> completionHandler);
+
+		[Export ("name")]
+		string Name { get; }
+
+		[Export ("monitoredIdentifiers")]
+		string [] MonitoredIdentifiers { get; }
+
+		[Export ("addConditionForMonitoring:identifier:")]
+		void AddCondition (CLCondition condition, string identifier);
+
+		[Export ("addConditionForMonitoring:identifier:assumedState:")]
+		void AddCondition (CLCondition condition, string identifier, CLMonitoringState state);
+
+		[Export ("removeConditionFromMonitoringWithIdentifier:")]
+		void RemoveCondition (string identifier);
+
+		[Export ("monitoringRecordForIdentifier:")]
+		[return: NullAllowed]
+		CLMonitoringRecord GetMonitoringRecord (string identifier);
+	}
+
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface CLLocationUpdater {
+		[Static]
+		[Export ("liveUpdaterWithQueue:handler:")]
+		[return: NullAllowed]
+		CLLocationUpdater Create (DispatchQueue queue, Action<CLUpdate> handler);
```

Comment:
Swift name is a bit more descriptive:

```suggestion
		CLLocationUpdater CreateLiveUpdates (DispatchQueue queue, Action<CLUpdate> handler);
```